### ISSUE TITLE
Pie chart with background color type Brush

### ISSUE THREAD

**Issue by antocara** [ehsannarmani/ComposeCharts#5]
Awesome job, would it be possible to add support so that pie chart can have a background color type Brush like Bars Charts? Thank you

**Comment by ehsannarmani** [comment]
@antocara Hi dear, you want to set brush color for pie chart pieaces?

**Comment by antocara** [comment]
Yep,  I would like to be able to put a gradient on the background of each pie

**Comment by ehsannarmani** [comment]
@antocara Sure, ill plan to do that.

**Comment by antocara** [comment]
Thank you so much
